workers/ohsome_quality_analyst/utils/definitions.py

```diff
@@ -211,5 +210,26 @@ def get_report_classes() -> Dict:
     )
 
 
-ReportResult = collections.namedtuple("Result", "label value text")
-ReportMetadata = collections.namedtuple("Metadata", "name description")
+def get_indicator_names() -> List[str]:
+    return list(load_metadata("indicators").keys())
+
+
+def get_report_names() -> List[str]:
+    return list(load_metadata("reports").keys())
+
+
+def get_layer_names() -> List[str]:
+    return list(load_layer_definitions().keys())
+
+
+def get_dataset_names() -> List[str]:
+    return list(DATASETS.keys())
+
+
+def get_fid_fields() -> List[str]:
+    fid_fields = []
+    for _, dataset in DATASETS.items():
```

Comment:
```suggestion
    for dataset in DATASETS.values():
```

Reply:
I used a helper function to flatten the dict.